relay.go

```diff
@@ -398,7 +398,7 @@ func (r *Relayer) handleCallReq(f lazyCallReq) error {
 		return nil
 	}
 
-	call, err := r.relayHost.Start(f, r.relayConn)
+	call, err := r.relayHost.Start(&f, r.relayConn)
```

Comment:
should we be converting to a pointer when the `lazyCallReq` is creatd? that way it'll be consistent (pointer everywhere) vs right now the `handleCallReq` method is using a value

Reply:
we only need to mutate the `lazyCallReq` in `Start()`, so is it a good idea to pass pointers everywhere?

Reply:
Generally better to be consistent (e.g., even our read-only methods take a pointer, even though a value is enough).